src/main/java/com/android/volley/VolleyLog.java

```diff
@@ -88,8 +88,8 @@ private static String buildMessage(String format, Object... args) {
         // Walk up the stack looking for the first caller outside of VolleyLog.
         // It will be at least two frames up, so start there.
         for (int i = 2; i < trace.length; i++) {
-            Class<?> clazz = trace[i].getClass();
-            if (!clazz.equals(VolleyLog.class)) {
+            String clazz = trace[i].getClassName();
+            if (!clazz.equals(VolleyLog.class.getName())) {
```

Comment:
Using the class name instead of the class should be reasonable, but the one caveat is that Class#getName() uses reflection, which we should aim to minimize on a potentially hot code path - and this is a loop inside of a logging method.

This cost can be minimized if we fetch the class name in a static constant, e.g.

`private static final String CLASS_NAME = VolleyLog.class.getName()`

and reference it here so it's loaded once per process instead of on every iteration of this loop.